core/src/main/java/org/infinispan/factories/EmptyConstructorFactory.java

```diff
@@ -27,31 +23,26 @@
  * @since 4.0
  */
 
-@DefaultFactoryFor(classes = {InboundInvocationHandler.class, RemoteCommandsFactory.class, ExternalizerTable.class,
-                              BackupReceiverRepository.class, CancellationService.class, TimeService.class})
+@DefaultFactoryFor(classes = {BackupReceiverRepository.class, CancellationService.class, ExternalizerTable.class,
+                              InboundInvocationHandler.class, RemoteCommandsFactory.class, TimeService.class})
```

Comment:
Nitpicking here a bit, but it seems there are a lot of extra additions and removals in this file, when it seems we only needed some removals and 1 addition.  There was a lot of reordering of code, which afaik isn't really necessary.